docs/pages/messages/missing-root-task-in-turbo-json.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Missing root task in turbo.json
+description: Missing root task in turbo.json error
+---
+
+# Missing root task in turbo.json
+
```

Comment:
This is incomplete. I'm a little stuck with explaining topological dependencies and why the root package is always an implicit dependency in a monorepo (cc @anthonyshew)